msteams-platform/graph-api/import-messages/import-external-messages-to-teams.md

```diff
@@ -147,7 +147,8 @@ HTTP/1.1 202 Accepted
 After the team and channel have been created, you can begin sending back-in-time messages using the `createdDateTime`  and `from`  keys in the request body. **NOTE**: messages imported with `createdDateTime` earlier than the message thread `createdDateTime` is not supported.
 
 > [!NOTE]
-> createdDateTime must be unique across messages in the same thread.
+> * `createdDateTime` must be unique across messages in the same thread.
+> * `createdDateTime` supports timestamps with milliseconds precision. (For example, if the incoming request message has `createdDateTime` 2020-09-16T05:50:31.0025302Z it would be converted to 2020-09-16T05:50:31.002Z when it is ingested.)
```

Comment:
2Z, then it would be converted to...

Reply:
@animeshkundu - please incorporate AkJo's comments. In addition, please verify and commit the following suggestion.
```suggestion
> * `createdDateTime` supports timestamps with milliseconds precision. For example, if the incoming request message has `createdDateTime` 2020-09-16T05:50:31.0025302Z it would be converted to 2020-09-16T05:50:31.002Z when it is ingested.
```